infra/modules/providers/azure/app-monitoring/README.md

```diff
@@ -0,0 +1,92 @@
+# Azure Monitor
+
+This custom Terraform based `app-monitoring` module deploys the "Azure Monitor" service in order to offer visibility into the behavior of your deployed Azure Resources. This module is recommended for monitoring Azure Resources deployed alongside App Services (ex. App Service Plan, App Gateway, VM, etc.). For direct monitoring of an App Service, the [app-insights](https://github.com/microsoft/cobalt/tree/master/infra/modules/providers/azure/app-insights) module is instead recommended. That module leverages an "Azure Monitor" service called [application insights](https://www.terraform.io/docs/providers/azurerm/r/application_insights.html).
+
+This module is used to map configurable `alert criteria` to a list of Azure Resources for monitoring. `alert criteria` for Azure Resources comes in two forms, `metrics` and `logs`. This module introduces `metrics` based monitoring only. As a result, this module offers configurable `metric alert criteria`. The way `metric alert criteria` is configured depends entirely on the `metrics` options offered by the Azure Resources selected for monitoring. (ex. The App Service Plan resource offers 'CpuMonitoring' as a configurable `metric`.)
+
+In addition to the `metric alert criteria`, this module introduces a configurable `action group` that when paired with `metric alert criteria` can be configured to trigger events like e-mail notifications.
+
+#### More on Azure Monitoring
+
+"Azure Monitor maximizes the availability and performance of your applications by delivering a comprehensive solution for collecting, analyzing, and acting on telemetry from your cloud and on-premises environments. ..All data collected by Azure Monitor fits into one of two fundamental types, metrics and logs. Metrics are numerical values that describe some aspect of a system at a particular point in time. They are lightweight and capable of supporting near real-time scenarios. Logs contain different kinds of data organized into records with different sets of properties for each type." - Source: [Azure Monitor Overview](https://docs.microsoft.com/en-us/azure/azure-monitor/overview)
+
+The list of `metrics` available for configuration depends entirely on the Azure Resource chosen to be monitored. Please visit [Supported Metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported) for a complete list of metrics supported by the Azure Monitor service.
+
+#### More on Terraform Resources
```

Comment:
`Dependent Terraform Resources` may be more descriptive of the content below

Reply:
Terraform Resources are now under a sub-section titled **Definition**.